index.js

```diff
@@ -722,7 +540,7 @@ app.post('/api/xethru', Validator.body(['locationid', 'state', 'rpm', 'mov_f', '
 
       const location = await db.getLocationData(locationid)
       if (location.isActive && !location.heartbeatSentAlerts) {
-        await handleSensorRequest(location, RADAR_TYPE.XETHRU)
+        await StateMachine.getNextState(location, handleAlert)
```

Comment:
I think this worked out well passing in the `handleAlert` function. Good way to factor out the StateMachine.js file but not need to completely tear apart `index.js` yet.